### ISSUE TITLE
Add formatting 

### ISSUE THREAD

**Issue by gaetschwartz** [nushell/vscode-nushell-lang#131]
It would be amazing if the extension could also be a formatter for nu files. Is it something planned ? If it is already the case I couldn't find any documentation for it ?

**Comment by fdncred** [comment]
Agreed. Which is why we are working on https://github.com/fdncred/nufmt. If it matures enough we'll move it to the nushell org and use it from there perhaps.

**Comment by AbdulSamadMalik** [comment]
+1

**Comment by fdncred** [comment]
I've been using this recently and it seems to work great!
https://github.com/blindFS/topiary-nushell

**Comment by AlekSi** [comment]
Is it better than nufmt?

**Comment by fdncred** [comment]
yes

**Comment by ai-zelenin** [comment]
+1

**Comment by argh-tom** [comment]
waiting for the formatter to come to you from fish 😄
